sources/dicom/large_image_source_dicom/assetstore/dicomweb_assetstore_adapter.py

```diff
@@ -192,25 +339,33 @@ def importData(self, parent, parentType, params, progress, user, **kwargs):
 
             # Set the DICOMweb metadata
             item['dicomweb_meta'] = get_dicomweb_metadata(client, study_uid, series_uid)
-            item = Item().save(item)
-
-            # Create a placeholder file with the same name
-            file = File().createFile(
-                name=f'{series_uid}.dcm',
-                creator=user,
-                item=item,
-                reuseExisting=True,
-                assetstore=self.assetstore,
-                mimeType=None,
-                size=0,
-                saveFile=False,
-            )
-            file['dicomweb_meta'] = {
+            item['dicom_uids'] = {
                 'study_uid': study_uid,
                 'series_uid': series_uid,
             }
-            file['imported'] = True
-            File().save(file)
+            item = Item().save(item)
+
+            instance_results = client.search_for_instances(study_uid, series_uid)
+            for instance in instance_results:
+                instance_uid = instance[instance_uid_key]['Value'][0]
+
+                file = File().createFile(
+                    name=f'{instance_uid}.dcm',
+                    creator=user,
+                    item=item,
+                    reuseExisting=True,
+                    assetstore=self.assetstore,
+                    mimeType='application/dicom',
+                    size=0,
```

Comment:
If we set the size to `None` rather than `0`, the Girder UI won't print any size, rather than the misleading `0`.

Reply:
Fixed!